apps/transport/lib/transport_web/controllers/resource_controller.ex

```diff
@@ -128,6 +128,15 @@ defmodule TransportWeb.ResourceController do
     end
   end
 
+  defp get_filename(headers, url) do
```

Comment:
J'hésite à introduire un helper générique (module à part) ici avec 2 méthodes:
- extract_from_header (retournant `nil` si pas de match)
- extract_from_url (voir commentaire plus bas)

Qu'on appellerait ensuite en faisant: `helper.extract_from_header(...) || helper.extract_from_url`.

Ca permettrait:
1. De mettre ces deux méthodes sous test (il peut y avoir des cas un peu pourris cf plus bas)
2. De la réutiliser dans la codebase (on fait ce même parsing à d'autres endroits)
3. Dans le futur, de supporter des choses comme `filename*` si ça commence à apparaître (https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Disposition)

Je me suis demandé si quelque chose de built-in ne faisait pas déjà ça dans plug ; apparemment oui mais non (https://github.com/elixir-plug/plug/blob/master/src/plug_multipart.erl#L29 mais privé).